src/styles/Typography.js

```diff
@@ -1,23 +1,15 @@
-// import { createGlobalStyle } from 'styled-components';
-// import st from '../assets/fonts/ShareTech-Regular.ttf';
-// import ys from '../assets/fonts/YoungSerif-Regular.ttf';
+import { createGlobalStyle } from 'styled-components';
+import TajawalRegular from '../assets/fonts/Tajawal/Tajawal-Regular.ttf'
 
-// const Typography = createGlobalStyle`
-//   @font-face {
-//     font-family: "Share Tech";
-//     src: url(${st});
-//   }
-//   @font-face {
-//     font-family: "Young Serif";
-//     src: url(${ys});
-//   }
-//   html {
-//     font-family: 'Share Tech', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-//   }
-//   h1, h2, h3 {
-//     font-family: 'Young Serif', -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-//   }
+const Typography = createGlobalStyle`
+  @font-face {
```

Comment:
Because Tajawal is **not** a [variable font](https://web.dev/variable-fonts/), we need to write a @font-face declaration for each weight used: regular, normal, and bold. Otherwise, the browser simulates the font-weight and it doesn't look as balanced as the designed font weight.

Suggested code: 
`@font-face {
font-family: "Tajawal";
src: url(${TajawalRegular});
font-weight: normal;
}

@font-face {
font-family: "Tajawal";
src: url(${TajawalMedium});
font-weight: 500;
}

@font-face {
font-family: "Tajawal";
src: url(${TajawalBold});
font-weight: bold;
}
`

Reply:
@liminalfun 

Good to know!  Are Regular, Medium & Bold sufficient for our needs?

Reply:
@Carms-dev 

Regular (400), Medium (500), and Bold (700) are sufficient for the design! 

I don't foresee any uses for ExtraLight, Light, ExtraBold, or Black.

For future reference, Black is useful for text overlays on image backgrounds. However, we don't use that pattern in our design, so it's not necessary to include that weight for this project.

Reply:
Great explanation!  I have removed those font files accordingly.  Note that Lato doesn't have medium (500) font-weight!